### PR TITLE
Add a Contributing guide

### PR DESCRIPTION
**Pull request by SpaceXCheeseWheel**
Unsure of what category it should go in, so I have put it into "Common" for now.

### CODE REVIEW

File: docs/common/contributing-guide.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Contributing to Docs
+description: How you can help write documentation.
+toc_max_heading_level: 5
+---
+# Contributing to Docs
+Thank you for your intrest in helping out!
+
+## Getting started
+This project uses Docusarus to provide documentation. You may either edit the documentation directly using the GitHub editor in your browser or edit them locally. If you edit the files locally, you can preview your changes live before pushing them to the remote repository. In order to preview your changes before pushing them to the repository, you will need to set up a local development environment.
```
Comment (sulu5890) [inline docs/common/contributing-guide.md:10]:
```suggestion
PaperMC uses [Docusarus](https://docusaurus.io) to manage documentation. You may either edit the documentation directly using the GitHub editor in your browser or edit them locally. If you edit the files locally, you can preview your changes live before pushing them to the remote repository. In order to preview your changes before pushing them to the repository, you will need to set up a local development environment.
```

File: docs/common/contributing-guide.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Editing and Making pages
+Documentation is written using the standard
+[Markdown syntax](https://www.markdownguide.org/basic-syntax/). There are also advanced features
+added by Docusarus, you can find more [here](https://docusaurus.io/docs/markdown-features).
+
+Documentation pages are currently located in the "docs" directory. Each project has it's own directory. The final formatting and orginization of the files and directories is stil in flux, you can find more information in the `#docs` channel on Paper's [Discord server](https://discord.com/invite/papermc).
+To add your page to the sidebar, edit `sidebars.js`.
+
+## Discussion
+If you need any help making changes, or want to make a larger change, you can discuss it through the `#docs` channel on Paper's [Discord server](https://discord.com/invite/papermc).
```
Comment (sulu5890) [inline docs/common/contributing-guide.md:39]:
```suggestion
If you need any help making changes or want to make a larger change, you can discuss it through the `#docs` channel on the PaperMC [Discord server](https://discord.com/invite/papermc).
```

File: docs/common/contributing-guide.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+4. Run the following to start the local site
+   ```
+   yarn run start
+   ```
+   You will be able to access the local site by browsing to `http://localhost:3000/`. Any edits you make will update the page live.
+
+## Editing and Making pages
+Documentation is written using the standard
+[Markdown syntax](https://www.markdownguide.org/basic-syntax/). There are also advanced features
+added by Docusarus, you can find more [here](https://docusaurus.io/docs/markdown-features).
```
Comment (sulu5890) [inline docs/common/contributing-guide.md:33]:
```suggestion
[Markdown syntax](https://www.markdownguide.org/basic-syntax/). Additionally, Docusaurus natively [extends Markdown](https://docusaurus.io/docs/markdown-features) providing more advanced features.
```

**Review by sulu5890 — COMMENTED** [review]
Generally looks fine. Thanks for a start on this. I've made a few comments but nothing serious.

I'd like this at some point to include more details about how to install git, using something like vscode(?), how to create a fork, use a new branch, how to make a pull request etc. Although, that is by no means blocking this. I'm totally happy to merge this as-is and add more details (possibly even images) at a later time.

Something else I've considered is officially accepting new pages/significant changes over Discord in essentially any format (even a google doc or so) for people who want to add documentation but aren't familiar with git. But I'm not sure about that yet.

For some of the comments: PaperMC is the org, also home to velocity, waterfall, etc., Paper is the Minecraft server.

**Comment by SpaceXCheeseWheel** [comment]
I'll go ahead and add sections for git and vsc setup, but honestly I'm not too experienced in it. I myself am still more or less trying to figure out how to use this more effectively, so it's as much of a guide for myself as it is for others. As such, my concern is that it probably will not be following best practices, etc.

**Pull request closed without merging**